cmd/embedded-cluster/reset.go

```diff
@@ -365,11 +365,6 @@ var resetCommand = &cli.Command{
 			Usage: "Disable interactive prompts",
 			Value: false,
 		},
-		&cli.BoolFlag{
```

Comment:
is this only for testing purposes? or is this a proposed change? im not sure this is desirable

Reply:
Got it. I missed that. Makes sense now